apps/mobile-wallet/src/components/AddressesTokensList.tsx

```diff
@@ -103,6 +110,18 @@ const AddressesTokensList = ({ addressHash, isRefreshing, parentModalId }: Addre
           )
         )
       )}
+      {hiddenAssetIds.length > 0 && (
+        <HiddenAssetBtnContainer>
+          <Button
+            title={t(hiddenAssetIds.length === 1 ? 'nb_of_hidden_assets_one' : 'nb_of_hidden_assets_other', {
+              count: hiddenAssetIds.length
+            })}
```

Comment:
The right way to do this based on docs is:
```ts
t('nb_of_hidden_assets', { count })
```

basically without the `_one` or `_other` at the end.

but for some reason, this is not working...

https://www.i18next.com/translation-function/plurals

Reply:
I figured out what the problem was, I made a PR on top of yours that fixes it:
- https://github.com/alephium/alephium-frontend/pull/1154